Code/Framework/AzToolsFramework/AzToolsFramework/PaintBrushSettings/PaintBrushSettingsWindow.cpp

```diff
@@ -103,7 +88,7 @@ namespace PaintBrush
         // and this is only visible while in a painting component mode, so we don't ever need to disable the controls.
         viewOptions.isDisabledInComponentMode = false;
         // Default size of the window
-        viewOptions.paneRect = QRect(50, 50, 350, 145);
+        viewOptions.paneRect = QRect(50, 50, 350, 210);
```

Comment:
question: Maybe we could move this to an AZ_CVAR for easier tweaking if we expect it to change much more? Could use an AZ::Vector4 for it to store the dimensions potentially?

Reply:
I think a CVAR / AZ::Vector4 would be overkill for this. It's not something that needs to be arbitrarily tweaked, since it's just the initial pane size. Once a user adjusts it, the resized version is serialized out and these numbers never affect anything again without deleting a windows registry setting (I wish I was kidding). 

The only reason you're seeing these numbers change a lot in my PRs is because I keep adding new fields to the window as I add paintbrush functionality, so the default window size needs to keep growing. But there shouldn't be a need to adjust them when new fields _aren't_ being added.